### PR TITLE
[Portuguese] Translate guide/introduction.md

### PR DESCRIPTION
**Pull request by kr4t0z**

### CODE REVIEW

File: site/docs/pt/guide/introduction.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+É por isso que nós meio que esperamos que você saiba configurar seu sistema, ou pelo menos que saiba aprender sozinho.
+(Instalar Node.js _da forma correta_ é tão complicado que não cabe aqui nessa página.)
+
+Se você tá se sentindo confuso nesse momento, você deve deixar Node.js pra trás e usar o [Deno](#pré-requisitos-para-o-deno) no lugar.
+
+:::
+
+Ainda confiante?
+Ótimo!
+[Primeiros Passos](./getting-started#getting-started-on-node-js)! :robot:
```
Comment (kr4t0z) [inline site/docs/pt/guide/introduction.md:210]:
```suggestion
[Bora Começar](./getting-started#getting-started-on-node-js)! :robot:
```

File: site/docs/pt/guide/introduction.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
-Um bot do telegram é uma conta de usuário especial que é automatizada por um programa.
-Qualquer pessoa pode criar um bot no telegram, o único pré-requisito é que você saiba um pouco de programação.
+Um bot do Telegram é uma conta de usuário especial automatizada por um programa.
+Qualquer pessoa pode criar um bot do Telegram, o único pré-requisito é que você saiba um pouco de programação.
 
-> Se você já sabe como criar bots, avance para [Primeiros Passos](./getting-started)!
+> Se você já sabe como criar bots, avance para os [Primeiros Passos](./getting-started)!
 
-grammY é uma biblioteca que torna a criação de bots uma tarefa super simples.
+grammY é uma biblioteca que torna a criação desses bots uma tarefa bem simples.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:8]:
```suggestion
O grammY é uma biblioteca que torna a criação desses bots uma tarefa bem simples.
```

File: site/docs/pt/guide/introduction.md [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+Depois disso você vai poder executar seu código.
+Seu bot então vai funcionar, até você parar de rodar ele.
 
 Já tá quase acabando…
 
 ## Como Manter um Bot Rodando
 
 …a não ser que você esteja levando o seu projeto a sério.
-Se você parar o seu bot (ou desligar o seu computador), seu bot para de responder e não vai reagir a nenhuma mensagem.
+Se você parar o seu bot (ou desligar o seu computador), seu bot congela e não vai responder à nenhuma mensagem.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:27]:
```suggestion
Se você parar o seu bot (ou desligar o seu computador), seu bot congela e não vai responder a nenhuma mensagem.
```

File: site/docs/pt/guide/introduction.md [hunk 4]
```diff
@@ -52,94 +52,94 @@ Um celular                        Servidores do Telegram                 seu lap
...
 
 ## O Que o Grammy Faz Para Você
 
 Bots interagem com o telegram via requisições HTTP.
 Toda vez que seu bot envia ou recebe mensagens, requisições HTTP vão e voltam entre os servidores do Telegram e o seu servidor/computador.
 
-Em sua essência, o grammY implementa toda essa comunicação pra você, de modo que você pode simplesmente digitar `sendMessage` no seu código e a mensagem será enviada.
-Além disso, tem uma variedade de outras coisas úteis que o grammY faz que torna mais simples criar seu bot.
+Por debaixo dos panos, o grammY implementa toda essa comunicação pra você, de modo que você pode simplesmente digitar `sendMessage` no seu código e a mensagem será enviada.
+Além disso, tem uma variedade de outras coisas úteis que o grammY para deixar mais simples criar seu bot.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:64]:
```suggestion
Além disso, tem uma variedade de outras coisas úteis que o grammY faz para deixar mais simples criar seu bot.
```

File: site/docs/pt/guide/introduction.md [hunk 5]
```diff
@@ -52,94 +52,94 @@ Um celular                        Servidores do Telegram                 seu lap
...
 
 Na próxima seção, você vai criar um bot escrevendo o arquivo de texto que contém o código-fonte na linguagem de programação [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html).
-A documentação do grammY não vai te ensinar como programar, então esperamos que você aprenda sozinho.
-Mas lembre-se: Criar bots para Telegram com o grammY é na verdade uma boa forma de aprender a programar! :rocket:
+A documentação do grammY não vai te ensinar a programar, então esperamos que você aprenda sozinho.
+Mas lembre-se: Criar bots para Telegram com o grammY é na verdade uma ótima forma de aprender a programar! :rocket:
 
-::: tip Aprendendo a Programar
+::: tip Como Aprender a Programar
 Você pode começar a aprender TypeScript com o [tutorial oficial](<https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html>) escrito pelo time do TypeScript e, depois, continuar a partir daí.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:78]:
```suggestion
Você pode começar a aprender TypeScript com o [tutorial oficial](<https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html>) escrito pelo time do TypeScript e continuar a partir daí.
```

File: site/docs/pt/guide/introduction.md [hunk 6]
```diff
@@ -52,94 +52,94 @@ Um celular                        Servidores do Telegram                 seu lap
...
-(Se você vir pessoas falando _Node_, elas são apenas preguiçosas demais pra digitar o ".js", mas querem dizer a mesma coisa.)
+(Se você vir pessoas falando _Node_, elas só estão sendo preguiçosas demais pra digitar o ".js", mas querem dizer a mesma coisa.)
 
 > O resto dessa seção te ajuda a decidir entre essas duas plataformas.
-> Se você já sabe como usar, pule para os [pré-requisitos para Node.js](#prerequisites-for-node-js) ou [para Deno](#pré-requisitos-para-o-deno).
+> Se você já sabe o que quer usar, pule para os [pré-requisitos para Node.js](#pré-requisitos-para-o-node-js) ou [para Deno](#pré-requisitos-para-o-deno).
 
-Node.js é tecnologia mais velha e madura.
-Se você precisa se conectar a um banco de dados fora do comum ou outras coisas de baixo nível relacionadas ao sistema, muito provavelmente você vai conseguir fazer isso com Node.js.
+Node.js é uma tecnologia mais velha e madura.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:100]:
```suggestion
O Node.js é uma tecnologia mais velha e madura.
```

File: site/docs/pt/guide/introduction.md [hunk 7]
```diff
@@ -52,94 +52,94 @@ Um celular                        Servidores do Telegram                 seu lap
...
+Se você precisa se conectar a um banco de dados diferente ou outras coisas de baixo nível relacionadas ao sistema, muito provavelmente você vai conseguir fazer isso com Node.js.
 Deno é relativamente novo, então as vezes ainda falta suporte para algumas coisas avançadas.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:102]:
```suggestion
O Deno é relativamente novo, então as vezes ainda falta suporte para algumas coisas avançadas.
```

File: site/docs/pt/guide/introduction.md [hunk 8]
```diff
@@ -52,94 +52,94 @@ Um celular                        Servidores do Telegram                 seu lap
...
 Isso significa instalar algumas ferramentas.
 
 #### Preparando sua Máquina para Desenvolvimento
 
-[Instale o Deno](https://deno.com/manual/getting_started/installation#download-and-install), se já não fez isso ainda.
+[Instale o Deno](https://deno.com/manual/getting_started/installation#download-and-install), caso não tenha feito isso ainda.
 
-Você também precisa de um editor de texto que se encaixa bem com programação.
+Você também precisa de um editor de texto que funciona bem com programação.
 O melhor para o Deno é o Visual Studio Code, geralmente chamado de apenas VS Code.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:137]:
```suggestion
O melhor para o Deno é o Visual Studio Code, geralmente chamado apenas de VS Code.
```

File: site/docs/pt/guide/introduction.md [hunk 9]
```diff
@@ -201,7 +201,7 @@ Se você nunca fez isso antes, espere se meter em problemas bem confusos que sã
 É por isso que nós meio que esperamos que você saiba configurar seu sistema, ou pelo menos que saiba aprender sozinho.
 (Instalar Node.js _da forma correta_ é tão complicado que não cabe aqui nessa página.)
 
-Se você tá se sentindo confuso nesse momento, você deve deixar Node.js pra trás e usar [Deno](#prerequisites-for-deno) no lugar.
+Se você tá se sentindo confuso nesse momento, você deve deixar Node.js pra trás e usar o [Deno](#pré-requisitos-para-o-deno) no lugar.
```
Comment (roziscoding) [inline site/docs/pt/guide/introduction.md:204]:
```suggestion
Se você tá se sentindo confuso nesse momento, aconselhamos deixar o Node.js pra trás e usar o [Deno](#pré-requisitos-para-o-deno) no lugar.
```

**Review by roziscoding — APPROVED** [review]
A few adjustments but other than that looks great :D

File: site/docs/pt/guide/introduction.md [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+> Pule o resto dessa página se você já sabe como desenvolver uma aplicação em Node.js ou Deno a vá direto para os [primeiros passos](./getting-started).
+
+Aqui estão algumas coisas interessantes sobre programação---coisas que são essencias para programar, mas raramente explicadas porque muitos desenvolvedores acham que elas são óbvias.
+
+Na próxima seção, você vai criar um bot escrevendo o arquivo de texto que contém o código-fonte na linguagem de programação [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html).
+A documentação do grammY não vai te ensinar a programar, então esperamos que você aprenda sozinho.
+Mas lembre-se: Criar bots para Telegram com o grammY é na verdade uma ótima forma de aprender a programar! :rocket:
+
+::: tip Como Aprender a Programar
+Você pode começar a aprender TypeScript com o [tutorial oficial](<https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html>) escrito pelo time do TypeScript e continuar a partir daí.
```
Comment (kr4t0z) [inline site/docs/pt/guide/introduction.md:78]:
```suggestion
Você pode começar a aprender TypeScript com o [tutorial oficial](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) escrito pelo time do TypeScript e continuar a partir daí.
```

**Pull request merged**